### PR TITLE
Improve CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by curquiza**
Should be merged after #49

### CODE REVIEW

**Review by bidoubiwa — APPROVED** [review]

File: templates/CONTRIBUTING.md [hunk 1]
```diff
@@ -121,11 +92,39 @@ We don't follow any other convention, but if you want to use one, we recommend [
...
+
+### Automation to Rebase and Merge the PRs
+
+This project integrates a bot that helps us manage pull requests merging.<br>
+_[Read more about this](https://github.com/meilisearch/integration-guides/blob/master/guides/bors.md)._
+
+### Automated Changelogs
+
+This project integrates a tool to create automated changelogs.<br>
+_[Read more about this](https://github.com/meilisearch/integration-guides/blob/master/guides/release-drafter.md)._
```
Comment (eskombro) [inline templates/CONTRIBUTING.md:114]: This throws 404, I guess it is normal since #49 isn't merged, but URL seems to be well written ;)

**Review by eskombro — APPROVED** [review]
👌 LGTM

It makes it ligther for regular contributor, I think that's an important goal 🎉

File: templates/CONTRIBUTING.md (hunk 1 above)
Comment (curquiza) [reply templates/CONTRIBUTING.md:114]: I'm waiting for your review to merge it 😉

Comment (eskombro) [reply templates/CONTRIBUTING.md:114]: I was reviewing it as you wrote this 😂

**Comment by curquiza** [comment]
I took your suggestions into account! Thanks! 🙂

**Pull request merged**